ferrocene/doc/core-certification/src/safety-plan/scope.rst

```diff
@@ -0,0 +1,47 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Certification scope
+===================
+
+The core library shall be suitable to be used in safety applications according to following safety standards up to the specified safety level:
+
+.. list-table::
+   :align: left
+   :header-rows: 1
+
+   * - Safety Standard
+     - Safety Level
+   * - |iec_61508_ref|
+     - |iec_61508_sil|
```

Comment:
<img width="889" height="412" alt="Image" src="https://github.com/user-attachments/assets/7d9d763c-90cd-4cc8-8651-013147eff619" />

This is an invalid ref, please fix it and check the others.

Reply:
Fixed it, should have been iec_61508_core_sil